### DISCUSSION TITLE
Enquiry about virtual-lamp-lighting-ceremony

### DISCUSSION THREAD

**Discussion by Sudevnedumparambath** [aravindvnair99/AravindVNair99#3] [Q&A] [answered]
Bro how can I use the programe virtual lamp lighting ceremony.
First let me ask is there any problem in me using it.
If not can you tell me how to use the code

**Comment by aravindvnair99** [comment, marked as answer]
Hey @Sudevnedumparambath!

Thanks for reaching out. You can use code as you wish but do make sure to follow the [license](https://github.com/aravindvnair99/Virtual-lamp-lighting-ceremony/blob/main/LICENSE) terms. For every repository you find on GitHub, you can use the code as per the license terms of that particular repository. Different licenses are there.

**Reply by Sudevnedumparambath** [reply]
Oh thats nice . 
You know.. I am actualy kind of a beginner in things related to websites. I have the basics but still just starting. So if you have some spare time, can you give me a rough idea on what i should be doing to use it as i intend to.
Like.., 
   Let's say i want to use this for an virtual inaugration. Ofcourse i would have to change the images and resourcses. And then i should  
   get the website hosted right??and so on

can you help me with it ?

**Reply by aravindvnair99** [reply]
@Sudevnedumparambath Sure, I'll probably put up some instructions in the README file. I've been wanting to do it for a while now as more people started noticing the project, but couldn't find time. 😅

You can fork the repository, update images and names, configure Firebase and host on Firebase Hosting. That's ideally what you have to do. I'll detail more in the README shortly to help you better.

And feel free to let me know if you have any other queries. Thanks for your interest and I'm happy to help! :)
